projects/cashmere/src/lib/datepicker/calendar/calendar.component.ts

```diff
@@ -567,7 +567,7 @@ export class CalendarComponent implements AfterContentInit, AfterViewChecked, On
             let curMin = +this.minutes;
             curMin++;
             if (curMin > 59) {
-                curMin = 59;
+                curMin = 1;
```

Comment:
hey @WistyDagel - this should actually get set to 0 instead of 1 so that 3:59 loops to 3:00 instead of 3:01